frontend/src/console/components/NoticeBoard/NoticeBoard.tsx

```diff
@@ -194,15 +194,30 @@ const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
     </p>
   );
 
+  const staticExplanation = (
+    <div className="explanation-container">
+      <p>
+        <WrappedMessage id="explanation" messages={messages} />
+        <br />
+        <br />
```

Comment:
Please see the explanation of the `br` element usage in the [HTML spec](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element). I believe it falls into the second category in this scenario.

TL;DR: please change it to:
```html
</p>
<p>
```

Reply:
Aha! Thanks for that! Added as requested.